README.md

```diff
@@ -3,28 +3,6 @@
 HIPIFY is a set of tools that you can use to automatically translate CUDA source code into portable
 [HIP](https://github.com/ROCm/HIP) C++.
 
-## Table of contents
```

Comment:
@LisaDelaney Please confirm.

Reply:
We've already linked to the docs so there's no real need for a TOC in the readme...but mostly I removed this because the readme is in markdown and the file that was linked to is now RST, which makes the linking messier than it's worth dealing with (I'm slowly removing TOCs/extended content from readmes in favor of pointing user to the docs website)

Reply:
We've already linked to the docs so there's no real need for a TOC in the readme...but mostly I removed this because the readme is in markdown and the file that was linked to is now RST, which makes the linking messier than it's worth dealing with (I'm slowly removing TOCs/extended content from readmes in favor of pointing user to the docs website)